### ISSUE TITLE
ERROR  invalid options argument after upgrading 2.17.4 -> 2.18.0

### ISSUE THREAD

**Issue by seth100** [nuxt/nuxt#27892]
### Environment

```sh
------------------------------
- Operating System: Linux
- Node Version:     v20.15.0
- Nuxt Version:     2.18.0
- CLI Version:      3.12.0
- Nitro Version:    -
- Package Manager:  yarn@4.1.0
- Builder:          webpack
- User Config:      ssr, target, dev, env, head, css, cssModules, extractCSS, plugins, buildModules, modules, i18n, vuetify, axios, proxy, router, build
- Runtime Modules:  @nuxtjs/i18n@7.3.1, nuxt-leaflet@0.0.27, nuxt-matomo@1.2.4, @nuxtjs/robots@2.5.0
- Build Modules:    @nuxtjs/axios@5.13.6, @nuxtjs/proxy@2.1.0, @nuxtjs/dotenv@1.4.2, @nuxtjs/vuetify@1.12.3, @nuxtjs/eslint-module@3.1.0
------------------------------
```

### Reproduction

Just take a project with these packages:
```json
"dependencies": {
    "@mapbox/mapbox-gl-geocoder": "^5.0.2",
    "@mdi/font": "^7.4.47",
    "@nuxtjs/axios": "^5.13.6",
    "@nuxtjs/proxy": "^2.1.0",
    "@nuxtjs/robots": "2.5.0",
    "@nuxtjs/vuetify": "^1.12.3",
    "animate.css": "^4.1.1",
    "leaflet": "^1.9.4",
    "mapbox-gl": "^2.15.0",
    "material-design-icons-iconfont": "^6.7.0",
    "nuxt": "^2.18.0",
    "nuxt-leaflet": "^0.0.27",
    "nuxt-matomo": "^1.2.4",
    "vue2-leaflet": "^2.7.1"
  },
  "devDependencies": {
    "@nuxtjs/dotenv": "^1.4.2",
    "@nuxtjs/eslint-config": "^12.1.0",
    "@nuxtjs/eslint-module": "^3.1.0",
    "@nuxtjs/i18n": "^7.3.1",
    "cross-env": "^7.0.3",
    "eslint": "^8.57.0",
    "include-media": "^2.0.0",
    "lru-cache": "^7.18.3",
    "minify-css-string": "^1.0.0",
    "rfs": "^10.0.0"
  },
```
and run
```sh
yarn dev
```

### Describe the bug

Just upgraded all of my Nuxt projects from `2.17.4` to `2.18.0` and I got:

```sh
   ╭────────────────────────────────────────────────────╮
   │                                                    │
   │   Nuxt @ v2.18.0                                   │
   │                                                    │
   │   ▸ Environment: development (dev)                 │
   │   ▸ Rendering:   client-side                       │
   │   ▸ Target:      static                            │
   │                                                    │
   │   Listening: http://localhost:3000/app             │
   │                                                    │
   ╰────────────────────────────────────────────────────╯

ℹ Preparing project for development                                                                                                                                                                                                            12:51:54 PM
ℹ Initial build may take a while                                                                                                                                                                                                               12:51:54 PM
✔ Builder initialized                                                                                                                                                                                                                          12:51:54 PM
✔ Nuxt files generated                                                                                                                                                                                                                         12:51:54 PM

● Client █████████████████████████ emitting (95%)  
 vue-client-plugin


 ERROR  invalid options argument                                                                                                                                                                                                                12:52:17 PM

  at optsArg (node_modules/@nuxt/webpack/node_modules/mkdirp/lib/opts-arg.js:13:11)
  at Function.mkdirp (node_modules/@nuxt/webpack/node_modules/mkdirp/index.js:11:10)
  at node_modules/webpack/lib/Compiler.js:494:26
  at eval (eval at create (node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:17:1)
  at node_modules/@nuxt/webpack/dist/webpack.js:304:7
  at _next0 (eval at create (node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:13:1)
  at eval (eval at create (node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:28:1)
  at node_modules/html-webpack-plugin/index.js:303:11
  at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

### Additional context

_No response_

### Logs

_No response_

**Comment by seth100** [comment]
sorry it's duplicate of https://github.com/nuxt/nuxt/issues/27873

**Comment by danielroe** [comment]
this should now be resolved in [v2.18.1](https://github.com/nuxt/nuxt/releases/tag/v2.18.1).
